mediapicker/src/main/java/com/mediapicker/gallery/presentation/fragments/FolderViewFragment.kt

```diff
@@ -39,22 +38,40 @@ class FolderViewFragment : BaseGalleryViewFragment(), OnItemClickListener<PhotoA
 
     override fun setUpViews() {
         super.setUpViews()
-        adapter = GalleryFolderAdapter(context!!, listOfFolders = emptyList(), onItemClickListener = this)
+        adapter = GalleryFolderAdapter(
+            requireContext(),
+            listOfFolders = emptyList(),
+            onItemClickListener = this
+        )
+
+        folderRV = childView.findViewById(R.id.folderRV)
+        actionButton = childView.findViewById(R.id.actionButton)
```

Comment:
yes, but it need to create generic form of the binding passing from the BaseFragment, i think it is too overkill, since it only has 3 little derived fragment now. wdyt @rc-angadsingh @AnkurSamaryaOlxIndonesia? i just think using findViewId it enough for now

Reply:
I think we can use `findViewById`.